recipes/gdk-pixbuf/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "gdk-pixbuf" %}
+{% set version_maj = "2.36" %}
+{% set version = "2.36.9" %}
+{% set sha256 = "8013b271ff1a691514b5bbc9b99f6ed456422d4da4a721a9db0b783abe8e740a" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.xz
+    sha256: {{ sha256 }}
+    url: http://ftp.gnome.org/pub/gnome/sources/{{ name }}/{{ version_maj }}/{{ name }}-{{ version }}.tar.xz
+    patches:
+      - 0001-changed-perl-script-to-use-env.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc  # [linux]
+    - llvm  # [osx]
+    - python <3
+    - m4
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - glib >=2.48.0
```

Comment:
This should probably be pinned to the standard value: `glib 2.51.*`

https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L49